install.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+
+BIN_DIR=$(pwd)
```

Comment:
It would be great to only set this if it isn't set, so people can do:

```
BIN_DIR=/usr/bin curl -sLK https://raw.githubusercontent.com/dunglas/frankenphp/main/install.sh | sh
```

Reply:
Thanks

Added so user can do `export BIN_DIR=...` however still not setting `/usr/bin` as default path because it may require `sudo` as `cat install.sh | sudo sh`, because not always `/usr/bin` has the permissions for the user. 

And adding extra instructions to set `BIN_DIR` in the README kind of complicates this simple process. Although added provision so user can install elsewhere too. That is why, `pwd` keeps things simple and user has a choice to move it to which ever bin dir of user's PATH choice.

Verification as below:

Fail

```
╰─$ sudo cat install.sh| sh
Password:
Warning: Failed to open the file /usr/local/bin/frankenphp: Permission denied
curl: (23) Failure writing output to destination
```

Success

```
╰─$ cat install.sh| sudo sh
Password:
######################################################################## 100.0%
Installed successfully to: /usr/local/bin/frankenphp
```